src/theory/quantifiers/solution_filter.h

```diff
@@ -29,30 +29,43 @@ namespace theory {
 namespace quantifiers {
 
 /**
- * This class is used to filter solutions based on some criteria.
+ * This class is used to filter solutions based on logical strength.
```

Comment:
"on some criteria (e.g. logical strength)"?

Reply:
No, this class is now solely devoted to logical strength.